common/containers/Tabs/ScheduleTransaction/selectors/schedulingToggle.ts

```diff
@@ -0,0 +1,12 @@
+import { AppState } from 'reducers';
+import { getSchedulingToggle } from './fields';
+
+interface ICurrentSchedulingToggle {
+  raw: string;
+  value: boolean;
+}
```

Comment:
Any reason this does raw / value, and isn't just a boolean?

Reply:
I've removed raw property, but leaving it as value so it matches format of other fields (object).